content/docs/command-reference/plots/show.md

```diff
@@ -12,7 +12,8 @@ usage: dvc plots show [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
                       [targets [targets ...]]
 
 positional arguments:
-  targets               Metric files to visualize.
+  targets               Files to visualize (supports any file,
```

Comment:
sounds too explicit. Why do we change this. I think if explain in the `plots` concept itself that plots are not only `dvc.yaml` based people won't expect `dvc.yaml` to be present in the first.

Reply:
Oops. I approved this change in the core repo: https://github.com/iterative/dvc/pull/4620#pullrequestreview-497717870

Reply:
> Files to visualize (supports any file, even when not found as `plots` in `dvc.yaml`). Shows all plots by default.

So how about changing/expanding on "All plots defined in `dvc.yaml` are used by default." in the **Description**, and simplifying the above to just "Plot files to visualize (shows all plots defined in `dvc.yaml` by default)" ?

Reply:
yep, sounds good to me ... still a bit verbose, and `dvc.yaml` will be confusing people but I don't know how to do it better

Reply:
Updated the description for plots `show` and `diff`, `metrics` seem alright. Please ping me if its ok as is now, I'll prepare PR for help messages to `dvc`.

Reply:
@shcheklein, do you think its ok now?